docs/communicate-with-javascript.md

```diff
@@ -1442,6 +1461,7 @@ like with objects. But unlike objects, there is no need to add any attributes:
 
 ```ocaml
 (* Abstract type for `timeoutId` *)
+(* Explain that abstract type here is a super nicety of abstract types. “We could bind to integer, that would be correct, but by being abstract we ensure more safety and express intentionaly” *)
```

Comment:
There is a whole section for abstract types at the top of the document:

https://github.com/melange-re/melange-re.github.io/blob/b1c1d5e86b17cc64775986d0a067c4975e1aee54/docs/communicate-with-javascript.md?plain=1#L222

Do you think this addition should be added over there? To have all the info about abstract types upfront, and avoid having to include it in the examples to explain it.